src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
+
+  for (const key in robot) {
+    if (!result[robot[key]]) {
+      result[robot[key]] = key;
+    } else if (result[robot[key]]) {
```

Comment:
you can avoid this `else if` bc if you already checked that "you don't have a key" and it's false, it means that you have this key))